.changeset/wild-mangos-clap.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Are you sure this should be empty? Seeing new things in react-ui

Reply:
yes! it needs to be updated! It was initially for docs. I'll update it